x-pack/plugins/apm/server/lib/services/get_service_instances/get_service_instance_transaction_stats.ts

```diff
@@ -133,10 +135,10 @@ export async function getServiceInstanceTransactionStats({
             })),
           },
           throughput: {
-            value: count.value,
```

Comment:
Isn't this covered by an API test? If not it really should be.

Reply:
I've seen this calculation in many places now. Should we make a helper for it?

```ts
function getThroughput({start, end, value }) {
 const deltaAsMinutes = (end - start) / 60 / 1000;
 return value / deltaAsMinutes;
}
```

Reply:
>Isn't this covered by an API test? If not it really should be.

It is covered but the snapshot was also incorrect.